src/index.html

```diff
@@ -7,8 +7,61 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="#home">
+    <header class="header">
+      <a href="#home" class="logo_link">
+        <img
+        class="logo_img"
+        src="images\logo.png"
+        alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="/">
```

Comment:
According to BEM, it is correct to add 2 underscores, because with this syntax, you currently do not have an element in blocks, but only blocks whose names consist of two words. Fix all cases with BEM naming please
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="/">
```